dispatcher.c

```diff
@@ -279,37 +280,75 @@ static inline ssize_t
 lzread(z_strm *strm, void *buf, size_t sze)
 {
 	char *ibuf = strm->ibuf;
+	size_t srcsize, destsize, total;
```

Comment:
This is not wrong, but the used code style is to declare all variables on a single line.

Reply:
I don't quite follow; the variables here are on a single line. Did you mean that you would prefer these `size_t` variables to be on separate lines?

Reply:
I meant to put each variable declaration on a separate line, I wasn't very clear, sorry

Reply:
OK, done.